README.md

```diff
@@ -80,6 +82,21 @@ pagrant:
       - node_modules/*
 ```
 
+### Override Ansible vars
+
+Default configuration is optimal for common use but you are able to tweak the provisioning by passing certain vars that are used in Ansbile roles and tasks.
+You can override the values you want by defining the `extra_vars` section in `.vagrantuser`.
+
+E.g. `/cool_project/vagrant/.vagrantuser`:
+
+```yml
+pagrant:
+  extra_vars:
+    php_fpm_version: '7.2' # Change the PHP version. Default is '7.1'. Supported are 5.6, 7.0, 7.1, 7.2
```

Comment:
I'm not sure this would work with 5.6

Reply:
it says this in the role. Didn't test it with 5.6
you might need to redefine the modules too